content/docs/command-reference/pull.md

```diff
@@ -168,14 +171,13 @@ example-get-started/
 │   ├── data.xml
 │   ├── data.xml.dvc
 ...
-├── model.pkl
-└── train.dvc
+└── model.pkl
```

Comment:
Did you try these commands? `$ tree example-get-started/` doesn't seem to be correct here if we're already inside that dir.

Reply:
Yes this wouldn't actually work. It should only be `$ tree`. Thanks.

Reply:
Cool